### PR TITLE
Adding IAO 'publication' and NCIT 'Additive'

### PR DESCRIPTION
**Pull request by westonslaughter**
Hey Laurent! 

Here are a couple additions to start to make sure I am doing it right. I am trying to add the IAO 'publication' class with its children 'journal article' and 'publication about an investigation' as well as adding the NCIT 'Additive' as a single term, under a CHEBI term in the eNM hierarchy. 

Also, I was considering submitting a 'Nano Entity' class defined (roughly) as follows: "Any entity or collection of entities that can be relevantly measured in nanometers in at least one of its dimensions." Besides wanting to check the definition- I want to check in about a few questions- if I were to submit 'nano entity' as a subclass of 'material entity' and as a parent class of 'nanomaterial' and 'nanoparticle' would that break the hierarchy of eNM.owl? Would it replicate the NM and NP data? I know I've kind of asked this question already, but I wanted to double check.

Also, I imagine I cannot add the RDF for a new eNM class until we have located a full RDF file for eNM owl.

Sorry for lengthy communications as always, and, as always, thank you!

Best,

Wes

### CODE REVIEW

File: internal/wikidata.owl [hunk 1]
```diff
@@ -243,14 +244,14 @@
...
     </owl:Class>
-	
+
 	<!-- http://www.wikidata.org/entity/Q41550 -->
 
     <owl:Class rdf:about="http://www.wikidata.org/entity/Q41550">
       <rdfs:subClassOf rdf:resource="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C17564"/>
       <rdfs:label xml:lang="en">Organisation for Economic Cooperation and Development</rdfs:label>
-	  <npo:synonym>OECD</npo:synonym>
+	        <npo:synonym>OECD</npo:synonym>
```
Comment (egonw) [inline internal/wikidata.owl:253]: Ah, I ran into that one this weekend too :)

**Review by egonw — APPROVED** [review]
Looks good to me!

**Comment by laurent2207** [comment]
- if I were to submit 'nano entity' as a subclass of 'material entity' and as a parent class of 'nanomaterial' and 'nanoparticle' would that break the hierarchy of eNM.owl? 

No this would not break it. We can discuss these sort of changes if appropiate. 

- Would it replicate the NM and NP data?

I am not entirely sure if I understand your question, but any RDF that would use the terms nanomaterial and nanoparticle will stay the same (the unique identiefier does not change for these terms).

-  I imagine I cannot add the RDF for a new eNM class until we have located a full RDF file for eNM owl.

Yes you can see [Adding a term from an ontology that is not yet used](https://enanomapper.github.io/tutorials/Added%20ontology%20terms/README.html)

**Comment by laurent2207** [comment]
Hey Wes,

No worries! Just keep me updated on your progress. If there are any questions just feel free to ask ;)

**Comment by westonslaughter** [comment]
Hey Laurent,

Will do! Currently I have a function that performs the matching of terms
(using regex for now, not yet fuzzywuzzy) but it takes a lot of time (2-3
hours...), so I am working on improving performance- though at the end of
the day, hopefully it will only need to be run once. I am somewhat
concerned that it will be even slower if using fuzzywuzzy. In any case, I'm
optimistic, and hopefully we'll have something working well soon.

Best,

Wes

**Pull request merged**